data/sql/updates/pending_db_world/rev_1708782048020249700.sql

```diff
@@ -0,0 +1,122 @@
+-- [START] // DB Update for .reset items command
+
+/*---------------------------------
+
+	Command and associated help
+	
+-----------------------------------*/
+
+-- Always delete before insert to ensure script repeatability :
+-- One value per line to facilitate readability
+DELETE FROM `command` WHERE `name` IN(
+'reset items',
+'reset items equiped',
```

Comment:
```suggestion
'reset items equipped',
```
Typos still exist in the SQL, `equiped` -> `equipped`, `optionnal` -> `optional`, `selecter` -> `selected`.

Reply:
Thanks